web/wp-content/themes/dctx-v2/header.php

```diff
@@ -0,0 +1,72 @@
+<?php
```

Comment:
:no_entry_sign: **Error**: Filenames should be all lowercase with hyphens as word separators. Expected phpcs-scan-z1ctkn.php, but found phpcs-scan-Z1ctKN.php (*WordPress.Files.FileName.NotHyphenatedLowercase*).